core/utils/eth_client_utils.go

```diff
@@ -51,6 +51,16 @@ func BytesToQuorumThresholdPercentages(quorumThresholdPercentagesBytes []byte) e
 	return quorumThresholdPercentages
 }
 
+func WeiToEth(wei *big.Int) float64 {
```

Comment:
I see this is only used for metrics, can't the conversion be handled on the visualization side of those metrics?
This should also not be used for any other purpose other than metrics and logs, because this is a lossy conversion (floats are not good for handling money because of that).

Reply:
Alternatively, why don't we keep the big.Float? That *might* be lossless (but I'd need to check the docs).

Reply:
@Oppen I agree no one likes floats, but Prometheus' Gauge accepts only float64 type